src/Customer.java

```diff
@@ -0,0 +1,118 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Customer {
+
+    private String name;
+    private String surname;
+    private LocalDate dateOfBirth;
+    private String cpf;
+    private String email;
+    private String mainPhone;
+    private String secondPhone;
+    private Address address;
+    private static int total;
+
+
+    public Customer() {
+    }
+
+    public Customer(String name, String surname, LocalDate dateOfBirth, String cpf, String email, String mainPhone, String secondPhone, Address address){
+        Customer.total++;
```

Comment:
A ideia é saber quantos Customers foram criados durante a execução do programa? Se for, o que acontece se eu chamar sempre o construtor vazio?

Reply:
É sim essa ideia.
Então, na verdade eu só coloquei os construtores vazios porque precisará quando interligar com o banco. Para mim não faz sentido ter construtor vazio, porque assim eu não consigo passar nenhum parâmetro.

Reply:
E nesta situação você não contara seus customers?

Reply:
Eu até posso contar, mas ai vou contar um customer sem nome, sem nada?

Reply:
Sua ideia é contar objetos criados em memória? Se sim, não faz diferença se estiver sem nenhuma informação. Para o sistema o objeto foi criado em memória.
Explique para o que servirá este `Customer.total++;`